spec/integration/federation/receive_federation_messages.rb

```diff
@@ -0,0 +1,224 @@
+require "spec_helper"
+require File.dirname(__FILE__) + "/factories"
```

Comment:
If it is possible, I'd prefer to use factories shipped from the diaspora_federation gem, instead of copying it here.